src/gradient.h

```diff
@@ -10,6 +10,62 @@
 #ifndef GRADIENT_H
 #define GRADIENT_H
 
+/** Enumeration of Gradient type used for determine gradient function to use.
+ */
+typedef unsigned char GradientType;
+#define GRADIENT_TITLE_ACTIVE   1
+#define GRADIENT_TITLE          2
+#define GRADIENT_COUNT          3
+
+#define GRADIENT_HORIZONTAL        0
+#define GRADIENT_VERTICAL          1
+
+extern unsigned int gradients[GRADIENT_COUNT];
+
+#define InitializeGradients() (void)(0)
```

Comment:
It's not currently used, but I think it would be good to use this initialize method to set `gradients` to their default values. That way if you removed a setting from your configuration and restarted it would do the right thing.